src/main/java/org/broadinstitute/consent/http/models/DuosUser.java

```diff
@@ -0,0 +1,24 @@
+package org.broadinstitute.consent.http.models;
+
+/**
+ * This class is used to represent an authenticated user that exists as a User in the Consent
+ * system. It extends the AuthUser class which is not guaranteed to have a user associated with it.
+ */
+public class DuosUser extends AuthUser {
+
+  private final User user;
+
+  public DuosUser(AuthUser authUser, User user) {
+    super(authUser.getAuthToken(),
+        authUser.getEmail(),
+        authUser.getName(),
+        authUser.getAud(),
+        authUser.getUserStatusInfo());
```

Comment:
Another approach here would be to add a constructor for `AuthUser` that accepts `AuthUser`.